helx/charts/nginx/values.yaml

```diff
@@ -52,6 +52,7 @@ ingress:
   tls:
     # -- Values inserted into the TLS block come from SSL.nginxTLSSecret and service.serverName for backward compatibility
     enabled: true
+  admin_ip_whitelist: "172.25.0.0/16,152.54.0.0/22,152.54.8.0/23,152.54.6.0/24,152.54.4.0/23,152.54.15.132/32"
```

Comment:
Do we want to leave this out of the default values for helx instances that run outside Europa?

Traffic will only flow through the VPN when the destination IP is inside Europa (152.\*, 172.\*, and a few others). But if accessing e.g. eduhelx in GCP your source IP will just be the IP of your router/ISP.

Reply:
Yes, provided that we have a sensible place to document it for future reference.